aikau/src/main/resources/alfresco/lists/AlfList.js

```diff
@@ -1242,11 +1259,20 @@ define(["dojo/_base/declare",
        * [loadData]{@link module:alfresco/lists/AlfList#loadData}.
        *
        * @instance
+       * @param {object} [payload] The payload supplied when making the reload request.
+       * @param {string} [payload.focusItemKey] An item to focus on if it is in the data that is reloaded
        */
-      onReloadData: function alfresco_lists_AlfList__onReloadData() {
+      onReloadData: function alfresco_lists_AlfList__onReloadData(payload) {
```

Comment:
If this is a payload, it suggests a publication. If that is the case then the parameters should be defined against the event. If it's not the case then we should avoid calling it a payload to avoid confusion.